label_studio/io_storages/base_models.py

```diff
@@ -389,12 +393,26 @@ def _scan_and_create_links(self, link_class):
                     f'"Treat every bucket object as a source file"'
                 )
 
-            self.add_task(data, self.project, maximum_annotations, max_inner_id, self, key, link_class)
+            task = self.add_task(data, self.project, maximum_annotations, max_inner_id, self, key, link_class)
             max_inner_id += 1
 
             # update progress counters for storage info
             tasks_created += 1
 
+            # add task to webhook list
+            tasks_for_webhook.append(task)
+
+            # settings.WEBHOOK_BATCH_SIZE
```

Comment:
This comment could be more helpful (eg, explain to reader how remaining `tasks_for_webhook` at the end of the batch will be handled)

Reply:
Comment fixed.